Add redirects, sitemaps, and limits topics (#944)
commerce-docs/microsite-commerce-storefront: src/content/docs/setup/seo/index.mdx

Reviewer: Description too long.
```suggestion
Learn how Edge Delivery Services SEO and Adobe Commerce storefront metadata work together, plus where to find indexing and metadata guidance.
```

Reviewer: ```suggestion
## Two layers to coordinate

The first layer is the Edge Delivery Services platform, which renders and serves HTML. The second is your Commerce catalog, which supplies the URLs and metadata layered on top.
```

Reviewer: This suggestion was an ExL validation update -- headers should have text underneath them -- I think this update is more graceful and provides needed context that explains the header.

### Edge Delivery Services (platform)

> Edge Delivery Services (EDS) shapes the first HTML response. Document-based pages usually ship most of their text there, while many catalog product detail pages (PDPs) load details later for performance unless you publish server-side metadata or use prerender. Search engines and large language model (LLM) crawlers still rely on that first payload, so Adobe publishes shared SEO and GEO guidance on <Link href="https://www.aem.live/docs/seo-geo" text="AEM.live" />. GEO guidance sits alongside SEO on that page because some crawlers do not fully render every page and rely on that initial HTML payload.

Reviewer: ```suggestion
Edge Delivery Services (EDS) controls the initial HTML payload delivered to crawlers. Document-based pages generally include most content in that payload, whereas many catalog product detail pages (PDPs) defer product data loading for performance unless prerendering or server-side metadata is enabled. Since search engines and large language model (LLM) crawlers may not fully render every page, Adobe publishes SEO and GEO guidance together on <Link href="https://www.aem.live/docs/seo-geo" text="AEM.live" /> to help ensure content remains discoverable.
```

Use these pages when you implement indexing, metadata, launch checks, or prerender for your storefront:

> - [SEO indexing](/setup/seo/indexing/) — canonical URLs, product URL format, redirects, hreflang, and multi-store considerations.

Reviewer: should hreflang be tagged `hreflang` since it's an attribute?

- [SEO metadata](/setup/seo/metadata/) — titles, descriptions, structured data, and PDP metadata workflows.
- [Sitemaps](/setup/seo/sitemaps/) — sitemap configuration for catalog pages, prerender integration, and multi-store hreflang.

> - [Platform limits](/setup/seo/platform-limits/) — URL naming rules, index size caps, and redirect limits that affect Commerce storefronts.

Reviewer: ```suggestion
- [Platform limits](/setup/seo/platform-limits/) — URL naming rules, index size caps, and redirect limits for Edge Delivery Services that affect Commerce storefronts.
```
